### DISCUSSION TITLE
Make prompts non-selectable in `console` code fences

### DISCUSSION THREAD

**Discussion by astrojuanlu** [pradyunsg/furo#284] [Ideas] [answered]
I noticed that the classic and alabaster themes add a `user-select: none;` property to `div.highlight span.gp` after https://github.com/sphinx-doc/sphinx/pull/6727 and https://github.com/sphinx-doc/sphinx/pull/9120, which I find quite nice. I wonder if furo could mimic this behavior?

**Comment by pradyunsg** [comment, marked as answer]
Hiya! Could you please file a [trackable task issue](https://github.com/pradyunsg/furo/issues/new?assignees=&labels=&template=issue.yml) for this? This certainly seems like something worth doing!

**Reply by astrojuanlu** [reply]
Done! https://github.com/pradyunsg/furo/issues/286
